automation_tools/__init__.py

```diff
@@ -1071,6 +1071,15 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             installer_options.update(execute(
                 setup_default_capsule, host=host, run_katello_installer=False
             )[host])
+        elif os.environ.get('INTERFACE'):
```

Comment:
I think this would be a problem if `os.environ.get('INTERFACE') would be TRUE , but it would never enter the `else:` part of the loop as we are checking for `os.environ.get('BRIDGE') is None`.

So I updated it accordingly by taking some hints from your suggestion.